OneCommunityGlobal/HighestGoodNetworkApp
# Prabh fix tasks tab responisveness

Task/ Issue: Make the Tasks tab AND edit a task’s details from that tab look and function well on phones.

I have to make the Tasks tab responsive to smaller screens. 

![image](https://user-images.githubusercontent.com/82662768/222896329-f384647b-b52c-480c-af3a-af55ccccd41e.png)

Before 

https://user-images.githubusercontent.com/82662768/222896562-56d9e6b3-5021-4c3a-9fae-0cb12d5e887e.mov

## Mainly changes explained:
Added some classes to <table> and its elements
Added media queries to make the tab responsive


## How to test:
check into the current branch
do `npm install` and `...` to run this PR locally
log as admin user
go to dashboard→ Tasks and Timelogs→ Tasks
check if the tasks tab is responsive. and work well on mobile.


## Note:
Lots of changes are made by Prettier: semicolon and indentation. 

TeamMemberTasks.jsx
In this file, my real changes are at lines 299 and 372. Rest are prettier changes you can ignore.
